.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: Build
```

Comment:
Could we call it "Code Analysis" or something like that? If we have other tools later we can add it to the same workflow.

Reply:
Good idea, done.

Reply:
Actually I was suggesting to use "Code Analysis" in line 1 so it will appear as one of the test categories here:
https://github.com/dogtagpki/jss/actions
Right now it's showing "Build" which doesn't really describe the jobs in this workflow.